ripple/src/main/webapp/businessdetails.js

```diff
@@ -197,14 +234,14 @@ function displayStarReviewSummary(avgStarRatings, numStarRatings) {
   }
 }
 
-/* Given an float, function rounds decimals to the tenths place. Ex: 3.14 -> 3.1 */
+/* Add a review: Given an float, function rounds decimals to the tenths place. Ex: 3.14 -> 3.1 */
 function RoundDecimals(float) {
   var roundedFloat = Math.round(float * 10) / 10;
   return roundedFloat;
   console.log("roundedFloat: " + roundedFloat);
 }
 
-/* If no reviews yet, default price display is N/A. 
+/* Add a review: If no reviews yet, default price display is N/A. 
```

Comment:
not sure how i understand price is related to review

Reply:
See comment above. These are functions that are all called when a user adds a review.

Reply:
Will rename. See comment above